napari/components/_viewer/view/main.py

```diff
@@ -101,6 +101,10 @@ def on_key_press(self, event):
         if layer is not None:
             layer.on_key_press(event)
 
+        if (event.text in self.viewer.key_bindings and not
+                event.native.isAutoRepeat()):
+            self.viewer.key_bindings[event.text](self.viewer)
```

Comment:
Do we want to be able to bind multiple functions to one key? If not, the previous statement should find a way to terminate the call prior to running this line.

Reply:
I will move it so that we overwrite the layer specific keybindings